xbmc/guilib/GUISpinControl.cpp

```diff
@@ -17,6 +17,11 @@
 #define SPIN_BUTTON_DOWN 1
 #define SPIN_BUTTON_UP   2
 
+namespace
+{
+constexpr float TEXT_SPACE = 5.0f;
```

Comment:
This variable name could use some documentation.

Reply:
done